pkg/helper/client.go

```diff
@@ -196,6 +199,30 @@ func ping(target string, timeout time.Duration) (string, error) {
 	})
 }
 
+func checkApiServerConnectivity(timeout time.Duration) error {
+
+	return wait.PollImmediate(1*time.Second, timeout, func() (bool, error) {
+		// Create new custom client to bypass cache [1]
+		// [1] https://github.com/operator-framework/operator-sdk/blob/master/doc/user/client.md#non-default-client
+		config, err := config.GetConfig()
+		if err != nil {
+			return false, errors.Wrap(err, "getting config")
+		}
+		// Since we are going to retrieve Nodes default schema is good
+		// enough
+		client, err := client.New(config, client.Options{})
+		if err != nil {
+			return false, errors.Wrap(err, "creating new custom client")
+		}
+		nodes := corev1.NodeList{}
```

Comment:
Nodes are huge, they contain list of all their pods, resources etc. We should get our own Enactment we created (or some meta object on kubernetes API, e.g. cluster version, if such a thing exists).

Reply:
We can try the counterpart of ```kubectl get namespace default```